### DISCUSSION TITLE
TuringCourses/math_phys/physics_experiment1/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#26] [General]
# TuringCourses/math_phys/physics_experiment1/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/math_phys/physics_experiment1/

**Comment by TonyCrane** [comment]
曾经（20、21 级）的部分建议：

- 王鲲老师比较注重有效数字，不注意的话给分会比你期望的略低；
- 陈水桥老师给分极好，实验 Ⅰ、Ⅱ 都是如此；
- 郭红丽老师人很好，会很耐心地讲实验，甚至愿意帮你调仪器，但是给分比较求是，好就是好，会给高，不好就是不好，会给很低；
- 郑远老师给分一般都比较好，之前做的是他的演示实验，基本给得都比较高。
